meta/main.yml

```diff
@@ -1,20 +1,72 @@
 # SPDX-License-Identifier: MIT
 ---
 galaxy_info:
-  author: John Doe <jdoe@corp.com>
-  description: Basic template for Linux system roles
-  company: John Doe, Inc.
+  # Replace with role's author name:
+  author: Joerg Kastning <jkastning@my-it-brain.de>
+  # Replace with the real description of what is role's purpose:
+  description: Install, configure and operate AIDE
+  # Replace with the company the role's author is member of:
+  company: Red Hat Inc.
+
+  # If the issue tracker for your role is not on github, uncomment the next
+  # line and provide a value
+  # issue_tracker_url: http://example.com/issue/tracker
+
+  # Some suggested licenses:
+  # - BSD (default)
+  # - MIT
+  # - GPLv2
+  # - GPLv3
+  # - Apache
+  # - CC-BY
   license: MIT
+
   min_ansible_version: "2.9"
+
+  # Optionally specify the branch Galaxy will use when accessing the GitHub
+  # repo for this role. During role install, if no tags are available, Galaxy
+  # will use this branch. During import Galaxy will access files on this
+  # branch. If Travis integration is configured, only notifications for this
+  # branch will be accepted. Otherwise, in all cases, the repo's default branch
+  # (usually main) will be used.
+  # github_branch:
+
+  #
+  # platforms is a list of platforms, and each platform has a name and a list
+  # of versions.
+  #
+  # platforms:
+  # - name: Fedora
+  #   versions:
+  #   - all
+  #   - "25"
+  # - name: SomePlatform
+  #   versions:
+  #   - all
+  #   - "1.0"
+  #   - "7"
+  #   - "99.99"
   platforms:
+    # Replace the below with your platform list:
     - name: Fedora
       versions:
         - all
     - name: EL
       versions:
         - "9"
+        - "10"
```

Comment:
```suggestion
```
"10" is not supported - this is what ansible-lint is complaining about - this is why we have to use `el10` in `galaxy_tags`